motion/p5js/motion-without-motion/index.js

```diff
@@ -0,0 +1,55 @@
+  function setup() {
+      createCanvas(400,400);
+      noStroke();
+      colorMode(HSB, 360, 100, 100, 100);
+      background(49,63,100);
+      fill (348,63,100,20); 
+      
+      myEllipse (50,250);
+      myEllipse (162.5,375);
+      myEllipse (350,150);
+      
+      // - - - - - - Function - - - - - - 
+      function myEllipse (xe, ye) {
+      push();
```

Comment:
Does this actually have an effect? The `push` `pop` commands need to be called in context with `translate` or `rotate`. Please check it.